docs/services/endlessh.md

```diff
@@ -0,0 +1,70 @@
+# Endlessh
+
+[Endlessh-go](https://github.com/shizunge/endlessh-go) is a A golang implementation of endlessh, a ssh trapit. Installing it is powered by the [mother-of-all-self-hosting/ansible-role-endlessh](https://github.com/mother-of-all-self-hosting/ansible-role-endlessh) Ansible role.
```

Comment:
https://github.com/mother-of-all-self-hosting/ansible-role-endlessh does not exist.

Or would you like us to take your own role and put it under the MASH organization?

-----

"a SSH trapit" is a typo. Try "an SSH tarpit".

It could also link to https://nullprogram.com/blog/2019/03/22/

There are some more typos in this sentence "a A golang" -> "a Golang implementation".

"endlessh" could be a link to the original C implementation (https://github.com/skeeto/endlessh).

Reply:
Please take my role and put it under the MASH organization.